1.8/administration/id-and-access-mgt/iam-api.md

```diff
@@ -4,28 +4,28 @@ post_title: Authentication HTTP API Endpoint
 
 You can make external calls to HTTP API endpoints in your DC/OS cluster.
 
-You must first obtain an HTTP API token and then include it in your HTTP request.
+You must first obtain an authentication token and then include it in your HTTP request.
 
-HTTP API tokens expire after 5 days. You can view the expiration time in the ["exp" (Expiration Time) Claim](https://tools.ietf.org/html/rfc7519#section-4.1.4) of the JSON Web Token (JWT). You can refresh your token by re-logging in to DC/OS.
+Authentication tokens expire after 5 days. You can view the expiration time in the ["exp" (Expiration Time) Claim](https://tools.ietf.org/html/rfc7519#section-4.1.4) of the JSON Web Token (JWT). You can refresh your token by re-logging in to DC/OS.
 
-# Generate the HTTP API token
+# Obtain an authentication token
 
-You can obtain your HTTP API token using DC/OS CLI. When you log into the DC/OS CLI, you paste an authentication token into your terminal prompt. This authentication token logs you into DC/OS CLI but does not authenticate you to the HTTP API endpoints. Complete the following steps to obtain your HTTP API token.
+You can obtain your authentication token using DC/OS CLI. When you log into the DC/OS CLI, you paste an OpenID Connect ID token into your terminal prompt. This OpenID Connect ID token logs you into DC/OS CLI but does not allow you to access the HTTP API endpoints. You must obtain an authentication token to gain access to the HTTP API endpoints. Complete the following steps to obtain your authentication token.
```

Comment:
"using the DC/OS CLI" instead of " using DC/OS CLI"? (I realize this has been sitting there before, but I think it's a grammar error, right?)